apiserver/params/network.go

```diff
@@ -135,8 +135,9 @@ type NetworkConfig struct {
 // DeviceBridgeInfo lists the host device and the expected bridge to be
 // created.
 type DeviceBridgeInfo struct {
-	HostDeviceName string `json:"host-device-name"`
-	BridgeName     string `json:"bridge-name"`
+	HostDeviceName       string `json:"host-device-name"`
+	BridgeName           string `json:"bridge-name"`
+	BondReconfigureDelay int    `json:"bond-reconfigure-delay"`
```

Comment:
I feel like this should just be "ReconfigureDelay" and the script we run should always just sleep that much.
Its just that we should have a different value if we know that the thing being bridged is a bond.